addons/point_of_sale/tests/test_frontend.py

```diff
@@ -1287,6 +1287,23 @@ def test_fiscal_position_two_tax_included(self):
         self.main_pos_config.open_ui()
         self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'FiscalPositionTwoTaxIncluded', login="accountman")
 
+    def test_tax_details_receipt(self):
```

Comment:
does this have to be a separate tour, or can it be integrated into one of the existing tours?

Reply:
I think I can manage to find a test to integrate it.